packages/ejson/package.js

```diff
@@ -1,10 +1,10 @@
 Package.describe({
   summary: "Extended and Extensible JSON library",
-  version: '1.0.13'
+  version: '1.0.14'
 });
 
 Package.onUse(function (api) {
-  api.use(['underscore', 'base64']);
+  api.use(['ecmascript', 'underscore', 'base64']);
   api.export('EJSON');
   api.export('EJSONTest', {testOnly: true});
   api.addFiles('ejson.js', ['client', 'server']);
```

Comment:
Done - I've also adjusted the package definition and structure a bit to better leverage `ecmascript`, and clean up the exports.